gridstatus/ercot.py

```diff
@@ -1773,6 +1777,41 @@ def _make_bid_curve(df):
 
         return self.parse_doc(df, verbose=verbose)
 
+    # @support_date_range("DAY_START") ??
```

Comment:
we can use `@support_date_range("DAY_START")` as is if we use the get document approach I mention below